This patch adds man file for the ppcppc64 compiler for ppc64 architecture.

Index: fpc/install/man/man1/ppcppc64.1
===================================================================
--- /dev/null
+++ fpc/install/man/man1/ppcppc64.1
@@ -0,0 +1,33 @@
+.TH ppcppc64 1 "30 may 1999" "Free Pascal" "Free Pascal Compiler"
+.SH NAME
+ppcppc64 \- Free Pascal Compiler (FPC) binary, name derived 
+from Portable Pascal Compiler
+
+.SH SYNOPSIS
+
+.B "ppcppc64 [options] [sourcefile]"
+.BR
+
+.SH DESCRIPTION
+This binary is the back-end binary of the
+.I Free Pascal Compiler (FPC)
+which is a
+.I Turbo Pascal
+and
+.I Delphi (7.0) 
+compatible standalone (non GCC frontend) multitarget Pascal compiler.
+
+This binary should not be called directly, instead the 
+.B fpc
+program should be used instead. 
+It will choose the right back-end for compiling units for the requested platforms.
+
+.SH OPTIONS
+
+For a complete list of all supported command-line options, see the
+.BR fpc (1)
+manual page.
+
+.SH SEE ALSO
+.BR  fpc (1)
+
